### DISCUSSION TITLE
Advance Examples

### DISCUSSION THREAD

**Discussion by metaory** [AutomaApp/automa#120] [General]
It be great to have some more Advanced examples that showcase different data extraction / iteration,

**Comment by Kholid060** [comment]
I'm currently building some kind of marketplace where users can share their workflow, and I'll add more examples there. The size of the extension will be bigger if I add examples inside the extension

**Reply by xiaoxuetong** [reply]
Very good. Looking forward to it

**Reply by xiaoxuetong** [reply]
Wiki documentation is a little bit sparse, and it is difficult to find on the Internet, I hope to improve it, thank you

**Reply by Kholid060** [reply]
I'll move the documentation to the website when it's published

**Comment by welcome2025** [comment]
特别是JS的高级实例太少了，还是不太会用
